gather_istio

```diff
@@ -196,4 +199,4 @@ function main() {
   echo
 }
 
-main
+main "$@"
```

Comment:
Better to have it provided with a flag instead? This way we could add more parameters in the future, each one with its own flag. This flag could be repeatable, and invoked multiple times for multiple namespaces, or accept a comma-separated list of namespaces...

Reply:
I had considered that option, but came to the conclusion that the names of the SMCP namespaces are the most important parameter. Adding a flag would hide that fact as soon as we introduce other flags. And this way, we keep the code simple for now. If you think we really should add this as a flag, I'll do it. I just don't see a real need for it.

Reply:
I'd prefer the flags for the reasons above. Keeping the code simple is not a strong argument, Usability and consistency is. I can approve this anyway, I don't want to be the only one blocking this pr :)